src/js/structs/NodesList.js

```diff
@@ -27,7 +25,7 @@ module.exports = class NodesList extends List {
 
       // Marathon API
       if (filters.ids) {
-        hosts = _.filter(hosts, function (host) {
+        hosts = hosts.filter(function (host) {
           return this.ids.indexOf(host.id) !== -1;
         }, {ids: filters.ids});
```

Comment:
did it in this commit: https://github.com/dcos/dcos-ui/pull/7/commits/d1e422a2da5b905d7ccbe88696ab099fd97ed018